### ISSUE TITLE
Writing video is not working. Black screen appears in the full video.

### ISSUE THREAD

**Issue by NaumanLiaqat** [pyannote/pyannote-video#33]
#### Description
 Every command is running fine. It is writing the video in the .final.mp4 file but when I play the video, it's just black screen with audio. It's only working on the sample video used in the tutorial. I've tried it on 10 videos but it is writing black screen instead of video.
Kindly help resolve this issue.

#### Steps/Code to Reproduce
pyannote-structure.py shot --verbose TheBigBangTheory3.mkv TheBigBangTheory3.shots.json

from pyannote.core.json import load_from
shots = load_from('TheBigBangTheory3.shots.json')
shots

pyannote-face.py track --verbose --every=0.5 TheBigBangTheory3.mkv TheBigBangTheory3.shots.json TheBigBangTheory3.track.txt

pyannote-face.py demo TheBigBangTheory3.mkv TheBigBangTheory3.track.txt TheBigBangTheory3.track.mp4

import io
import base64
from IPython.display import HTML
video = io.open('TheBigBangTheory3.track.mp4', 'r+b').read()
encoded = base64.b64encode(video)
HTML(data='''<video alt="test" controls><source src="data:video/mp4;base64,{0}" type="video/mp4" /></video>'''.format(encoded.decode('ascii')))

pyannote-face.py extract --verbose TheBigBangTheory3.mkv TheBigBangTheory3.track.txt ../../dlib-models/shape_predictor_68_face_landmarks.dat ../../dlib-models/dlib_face_recognition_resnet_model_v1.dat TheBigBangTheory3.landmarks.txt TheBigBangTheory3.embedding.txt

from pyannote.video.face.clustering import FaceClustering
clustering = FaceClustering(threshold=0.6)
face_tracks, embeddings = clustering.model.preprocess('TheBigBangTheory3.embedding.txt')
face_tracks.get_timeline()
result = clustering(face_tracks, features=embeddings)

pyannote-face.py demo TheBigBangTheory3.mkv TheBigBangTheory3.track.txt --label=TheBigBangTheory3.labels.txt TheBigBangTheory3.final.mp4

import io
import base64
from IPython.display import HTML
video = io.open('TheBigBangTheory3.final.mp4', 'r+b').read()
encoded = base64.b64encode(video)
HTML(data='''<video alt="test" controls><source src="data:video/mp4;base64,{0}" type="video/mp4" /></video>'''.format(encoded.decode('ascii')))

#### Expected Results
[MoviePy] >>>> Building video ../../pyannote-data/TheBigBangTheory.final.mp4
[MoviePy] Writing audio in TheBigBangTheory.finalTEMP_MPY_wvf_snd.mp3
100%|████████████████████████████████████████| 664/664 [00:01<00:00, 411.21it/s]
[MoviePy] Done.
[MoviePy] Writing video ../../pyannote-data/TheBigBangTheory.final.mp4
100%|████████████████████████████████████████▉| 752/753 [00:08<00:00, 87.43it/s]
[MoviePy] Done.
[MoviePy] >>>> Video ready: ../../pyannote-data/TheBigBangTheory.final.mp4

#### Actual Results
[MoviePy] >>>> Building video TheBigBangTheory3.track.mp4
[MoviePy] Writing audio in TheBigBangTheory3.trackTEMP_MPY_wvf_snd.mp3
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 736/736 [00:01<00:00, 434.07it/s]
[MoviePy] Done.
[MoviePy] Writing video TheBigBangTheory3.track.mp4
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1000/1000 [00:16<00:00, 62.41it/s]
[MoviePy] Done.
[MoviePy] >>>> Video ready: TheBigBangTheory3.track.mp4

#### Versions

absl-py==0.2.2
align==0.0.5
astor==0.7.0
backcall==0.1.0
bleach==1.5.0
boltons==18.0.0
boto==2.48.0
boto3==1.7.55
botocore==1.10.55
bz2file==0.98
certifi==2018.4.16
chardet==3.0.4
cloudpickle==0.5.3
cmake==3.11.0
colorama==0.3.9
cycler==0.10.0
dask==0.18.1
decorator==4.3.0
dlib==19.5.1
docopt==0.6.2
docutils==0.14
entrypoints==0.2.3
face==0.1.0
face-recognition-models==0.3.0
facenet==1.0.3
gast==0.2.0
gensim==3.5.0
grpcio==1.13.0
h5py==2.8.0
html5lib==0.9999999
idna==2.7
imageio==2.3.0
imutils==0.4.6
ipykernel==4.8.2
ipython==6.4.0
ipython-genutils==0.2.0
ipywidgets==7.3.0
jedi==0.12.1
Jinja2==2.10
jmespath==0.9.3
joblib==0.12.1
jsonschema==2.6.0
jupyter==1.0.0
jupyter-client==5.2.3
jupyter-console==5.2.0
jupyter-core==4.4.0
kiwisolver==1.0.1
Markdown==2.6.11
MarkupSafe==1.0
matplotlib==2.2.2
mistune==0.8.3
moviepy==0.2.3.4
munkres==1.0.12
nbconvert==5.3.1
nbformat==4.4.0
networkx==2.1
nltk==3.3
notebook==5.6.0
numpy==1.14.5
opencv-contrib-python==3.4.1.15
opencv-python==3.4.1.15
pandas==0.23.3
pandocfilters==1.4.2
parso==0.3.1
pickleshare==0.7.4
Pillow==5.2.0
prometheus-client==0.3.0
prompt-toolkit==1.0.15
protobuf==3.6.0
psutil==5.4.6
pyannote.algorithms==0.8
pyannote.core==1.4
pyannote.parser==0.7.1
Pygments==2.2.0
pyparsing==2.2.0
python-dateutil==2.7.3
pytz==2018.5
PyWavelets==0.5.2
pywinpty==0.5.4
pyzmq==17.1.0
qtconsole==4.3.1
requests==2.19.1
s3transfer==0.1.13
scikit-image==0.14.0
scikit-learn==0.19.1
scikit-video==1.1.10
scipy==1.1.0
Send2Trash==1.5.0
simplegeneric==0.8.1
simplejson==3.16.0
six==1.11.0
smart-open==1.6.0
sortedcollections==1.0.1
sortedcontainers==2.0.4
tensorboard==1.7.0
tensorflow==1.7.0
tensorflow-serving-api-python3==1.7.0
termcolor==1.1.0
terminado==0.8.1
testpath==0.3.1
toolz==0.9.0
tornado==5.1
tqdm==4.23.4
traitlets==4.3.2
urllib3==1.23
utils==0.9.0
wcwidth==0.1.7
web.py==0.40.dev1
Werkzeug==0.14.1
widgetsnbextension==3.3.0
xarray==0.10.7
xlrd==1.1.0

**Comment by hbredin** [comment]
Did you solve this issue? 
If not, can you share one of the videos for which it is not working?

**Comment by StonePanda** [comment]
@NaumanLiaqat  hello ,have u resolved this problem?

**Comment by StonePanda** [comment]
@hbredin hello, i met with this issue, and i find that when i changed the scale of my video from 1920×804 to 720×400, the issue disappered. i do not know why it works, hope u answer this. Thanks!!

**Comment by StonePanda** [comment]
@hbredin i found that i forgot to add the parameter --height which is the pixels height of demo video file. If the parameter was lower than the demo video height, i will get a wrong output video that is all black.
so if my demo video is 768×576， the command should be 
`pyannote-face.py demo demo.mkv demo_tracks.txt demo_output.mp4 --height=576`
hope it will work for others!
thanks for ur great work!

**Comment by hbredin** [comment]
Thanks for your feedback. Closing.
